packages/medusa/src/migrations/1671545490914-removed_staged_jobs.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class removedStagedJobs1671545490914 implements MigrationInterface {
+  name = "removedStagedJobs1671545490914"
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`DROP TABLE "staged_job"`)
```

Comment:
I would suggest not just removing the table and everything in it, if someone is upgrading to the new version it might have bad consequences of not handling the current events. I would suggest creating a script that moves the staged_jobs to be processed immediately. Maybe the script can be called in the up method?